source/changelogs/2021-08-01.md

```diff
@@ -0,0 +1,37 @@
+---
+title: August 2021 Changelog
+changelog: true
+description: August 2021 Pantheon changelog
+reviewed: "2021-08-02"
+---
+
+## Platform Improvements
+
+### Drupal 9.2.0
+
+[Drupal 9.2.2](https://www.drupal.org/project/drupal/releases/9.2.2) is now available for production sites. [Drupal 9](https://pantheon.io/docs/drupal-9) sites on Pantheon use [Integrated Composer](https://pantheon.io/docs/integrated-composer), enabling one-click core updates through the Dashboard. To check for available updates, click **Check Now** from the site Dashboard **Dev** tab.
+
+Drupal 9.2.x will receive security coverage until June 15, 2022. Sites on 9.1.x or earlier should update to [Drupal 9.1.11](https://www.drupal.org/project/drupal/releases/9.1.11).
+
+
+### Drupal 8.9.17
+
+[Drupal 8.9.17](https://www.drupal.org/project/drupal/releases/8.9.17) is now available on the Pantheon platform. Detailed information on applying and debugging core updates can be found in the [Core Updates documentation](/core-updates).
+
+
+### Drupal 7.82
+
+[Drupal 7.82](https://www.drupal.org/project/drupal/releases/7.82) is now available on the Pantheon platform. Pantheon is committed to providing Drupal 7 extended support for all customers through 2025, past Drupal 7’s end of life in 2022.
+
+
+### WordPress 5.8
+
+[WordPress 5.8](https://wordpress.org/news/2021/07/tatum/)[News – Releases – WordPress.org](https://wordpress.org/news/category/releases/) is available on the Pantheon platform. This release fixes security vulnerabilities, and users are urged to upgrade their sites immediately. Detailed information on applying and debugging core updates can be found in the [Core Updates documentation](/core-updates).
```

Comment:
Same on this one @joa-pan

Reply:
i think because the link is relativized. It should resolve when it builds.